package proto_ff;

import "yd_fieldoptions.proto";
import "Com.proto";

message tbFishAccountTable
{
  optional uint64 player_id = 1 [(yd_fieldoptions.db_field_type) = E_FIELDTYPE_PRIMARYKEY, (yd_fieldoptions.db_field_auto_increment)=true, (yd_fieldoptions.db_field_auto_increment_value)=10000, (yd_fieldoptions.db_field_comment) = "玩家ID"];
  optional string account = 2 [(yd_fieldoptions.db_field_type) = E_FIELDTYPE_UNIQUE_INDEX, (yd_fieldoptions.field_bufsize)=128, (yd_fieldoptions.db_field_bufsize)=128, (yd_fieldoptions.db_field_comment) = "账号"];
  optional string password = 3 [(yd_fieldoptions.db_field_type) = E_FIELDTYPE_INDEX, (yd_fieldoptions.field_bufsize)=128, (yd_fieldoptions.db_field_bufsize)=128, (yd_fieldoptions.db_field_comment) = "密码"];
  optional uint32 account_type = 4[(yd_fieldoptions.db_field_comment) = "账号类型"];
  optional string device_id = 5[(yd_fieldoptions.field_bufsize)=128, (yd_fieldoptions.db_field_bufsize)=128, (yd_fieldoptions.db_field_comment) = "设备ID"];
  optional uint64 phonenum = 6[(yd_fieldoptions.db_field_comment) = "手机号码"];
}

message tbFishPlayerData
{
  optional uint64 player_id = 1 [(yd_fieldoptions.db_field_type) = E_FIELDTYPE_PRIMARYKEY, (yd_fieldoptions.db_field_comment) = "玩家ID"]; //UserID用户ID
  optional string nickname = 2[(yd_fieldoptions.db_field_type) = E_FIELDTYPE_INDEX, (yd_fieldoptions.field_bufsize)=128, (yd_fieldoptions.db_field_bufsize)=128, (yd_fieldoptions.db_field_comment) = "昵称"];
  optional uint32 faceid = 3[(yd_fieldoptions.db_field_comment) = "头像"];
  optional uint64 regdate = 4[(yd_fieldoptions.db_field_comment) = "注册时间"]; //
  optional uint32 gender = 5[(yd_fieldoptions.db_field_comment) = "性别"]; //
  optional uint32 age = 6[(yd_fieldoptions.db_field_comment) = "年龄"];//
  optional uint64 phonenum = 8[(yd_fieldoptions.db_field_comment) = "手机号码"];
  optional uint64 jetton = 9[(yd_fieldoptions.db_field_comment) = "金币"];
  optional string ip = 11[(yd_fieldoptions.field_bufsize)=64, (yd_fieldoptions.db_field_bufsize)=64, (yd_fieldoptions.db_field_comment) = "IP地址"];
  optional uint64 last_login_time = 12[(yd_fieldoptions.db_field_comment) = "最近一次登录时间"];
  optional uint64 last_logout_time = 13[(yd_fieldoptions.db_field_comment) = "最近一次退出时间"];
}